tests/test_factories.py

```diff
@@ -93,9 +88,8 @@ def test_custom_page_streamfield():
     assert len(page.body) == 0
 
 
-@pytest.mark.skipif(
-    wagtail_version >= (4, 1),
-    reason="Cannot assign a naive list to a ListBlock value if using ReferenceIndex",
+@pytest.mark.skip(
+    reason="Cannot assign a naive list to a ListBlock value if using ReferenceIndex"
```

Comment:
question: is this needed at all? previously it was skipped for Wagtail >= 4.1 and now we only support Wagtail 4.1+ 🤔

Reply:
Well spotted. I've changed it to test assignment of a `ListValue` to the `StreamField` which still has some value I think